subprojects/gradle/gradle-extensions/src/main/kotlin/com/avito/Strings.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito
+
+import java.util.Locale
+
+/**
+ * replacement for deprecated kotlin's String.capitalize
```

Comment:
For the context:
https://kotlinlang.org/docs/whatsnew15.html#stable-locale-agnostic-api-for-upper-lowercasing-text

Reply:
added to comment